posts/2020-09-03-Planning-2021-Roadmap.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Planning the 2021 Roadmap"
+author: The Rust Core Team
+release: false
+---
+
+The core team is beginning to think about the 2021 Roadmap, and we want to hear from the community. We’re going to be running two parallel efforts over the next several weeks: the 2020 Rust Survey, to be announced next week, and a call for blog posts.
+
+Blog posts can contain anything related to Rust: language features, tooling improvements, organizational changes, ecosystem needs — everything is in scope. We would encourage you to try to identify themes or broad areas into which your suggestions fit in, because these help guide the project as a whole. We are trying something a bit different this year. When you write a post, please submit it into [this google form](https://forms.gle/Hv41uA5qJEY89XRm7) so we don’t miss it! We will try to look at posts not submitted via this form, too, but posts submitted here aren’t going to be missed. Any platform — from blogs to GitHub gists — is fine!
```

Comment:
Note to self: discoverability (`--doc-coverage`, `extern "C" fn f<T>`, `async {}` vs `async || {}`, `entry()`, docs.rs UI)